classes/Translator.php

```diff
@@ -136,6 +136,23 @@ public function loadLocaleFromRequest()
         return true;
     }
 
+    /**
+     * Sets the locale based on the browser accept languages
+     * @return bool
+     */
+    public function loadLocaleFromRequestLanguages()
+    {
+        $locales = Locale::listAvailable();
+
+        foreach(Request::getLanguages() as $request_lang) {
```

Comment:
add spaces after the foreach and if please, `$request_lang` to `$requestLang` please. Thanks!